ramp-frontend/ramp_frontend/views/ramp.py

```diff
@@ -823,14 +823,20 @@ def view_model(submission_hash, f_name):
                         submission, filename)
             )
 
+            workflow_element = WorkflowElement.query.filter_by(
+                name=filename.split('.')[0], workflow=event.workflow).one()
```

Comment:
Maybe define a separate helper function to get root ° extension from a path, and use it consistently? `os.path.splitext` is not doing what we need?